studio/pages/404.tsx

```diff
@@ -52,16 +49,16 @@ const Error404: NextPage = ({}) => {
           show404 ? 'opacity-100' : 'opacity-0'
         }`}
       >
-        <div className="w-[320px] flex flex-col items-center justify-center space-y-3">
-          <Typography.Title level={3}>Looking for something? 🔍</Typography.Title>
-          <Typography.Text className="text-center">
+        <div className="w-[320px] flex flex-col items-center justify-center space-y-3 text-center">
+          <Typography.Title style={{ fontSize: '1.5rem' }}>Looking for something? 🔍</Typography.Title>
```

Comment:
avoid specifying font sizes directly, use tailwind classes like text-lg or text-xl

Reply:
just a heads up as well, we're moving away from using the Typography component of our UI library

we'll just use vanilla html elements like h1 or h2 from here